Hands on1 final

ArturoAS opened the pull request:
Fixed these:

- The pull request is removing some files from a previous hands-on.
- The directory name must be "Group07" instead of "Grupo7".

rgcmme commented:
* The directory name is not correct yet.

ArturoAS commented:
Sorry, now it is

rgcmme commented:
Thanks, @dgarijo once assignment 4 is correct this pull request can be merged from my side.

dgarijo commented on Assignment4/ArturoAS-y160324/Task07.py:
```diff
@@ -0,0 +1,129 @@
+query = prepareQuery('''
+  SELECT DISTINCT ?x
+  WHERE{
+          ?x (rdfs:subClassOf/rdfs:subClassOf*) ns:Person
```
this is equivalent to rdfs:subClassOf+

dgarijo reviewed (changes requested):
Assignment is correct, but you need to remove all changes done to Notebooks and example1.ttl file

ArturoAS reviewed (commented):
Deleted the files that i modified

ArturoAS commented:
> Deleted the files that i modified

Sorry. I have no idea how to go back on the changes i made to the course materials directory so i just deleted the files that i modified: Task1,2,6,7 and the example1.ttl. Hope this can solve my merging problem. Wont happen again.
